public/data/imanishbarnwal.json

```diff
@@ -0,0 +1,22 @@
+{
+  "name": "Manish Kumar Barnwal",
+  "bio": "Engineer & Web Developer",
+  "avatar": "http://github.com/qwe123coder.png",
```

Comment:
This does not match your GitHub username?

Reply:
I forgot to change it now I fixed kindly check.

Reply:
Please do mark it as resolved after it is resolved